src/browser/disabled_frame_group_tracker.ts

```diff
@@ -323,7 +276,7 @@ export function addWindowToGroup(win: OpenFinWindow) {
                     } catch (error) {
                         writeToLog('error', error);
                     }
-                }, 16);
+                }, 30);
```

Comment:
Is there a particular reason for the timing change?

Reply:
Yes, just in testing it seemed that ~30 "fps" on the update was more than adequate for smooth resize so figured why not be a little kinder on the core.